res/controllers/Numark-Mixtrack-Pro-FX-scripts.js

```diff
@@ -141,9 +146,23 @@ MixtrackProFX.EffectUnit = function(deckNumber) {
     this.effectParam = new components.Encoder({
         group: "[EffectRack1_EffectUnit" + deckNumber + "_Effect1]",
         shift: function() {
+            if(MixtrackProFX.fxControlsMeta)
+            {
```

Comment:
{ should be on the same line, here and below
```suggestion
            if(MixtrackProFX.fxControlsMeta) {
```

Reply:
Are those the only problems from pre-commit? I'm on Windows and pre-commit has failed to build the environment correctly so am unable to run it locally.

Reply:
no, looks like pre-commit did complain only about other issues, maybe only relevant on 2.4 with ES7 (IIRC).
I think you should rebase onto 2.3, foce-push, then check the pre-coomit result ("Details" link in the CI box). There's also a patch which you can apply manually.

> I'm on Windows and pre-commit has failed to build the environment correctly so am unable to run it locally.

If you don't find any help online, check into the Zulip development stream and search for pre-commit on Windows. IIRC this issue comes up frequently.
https://mixxx.zulipchat.com/